lib/spack/spack/util/compression.py

```diff
@@ -167,7 +187,11 @@ def _unzip(archive_file):
     return extracted_file
 
 
-def _unZ(archive_file):
+def _system_unZ(archive_file):
+    """
+    Decompress UNIX compress style compression
+    Utilizes gunzip on unix and 7zip on Windows
+    """
     if sys.platform == "win32":
         result = _7zip(archive_file)
```

Comment:
Can you rename `_7zip` to `_system_7zip`.

Reply:
IMO the system part of 7zip should be implied considering no python module exists for it, but I'll make the change anyway.

Reply:
Thanks: I understand there is no module for it, but I think it is worthwhile to make it obvious with the method name.

Reply:
Very reasonable. Additionally I appear to have put my foot in my mouth as it turns out someone did actually make a 7zip python module and put it on pypi. Not a standard lib, but I'm still very incorrect.